content/en/docs/project-user-guide/custom-application-monitoring/get-started/monitor-sample-web.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Monitor Sample Web"
+keywords: 'monitoring, prometheus, prometheus operator'
+description: 'Monitor Sample Web'
+
+linkTitle: "Monitor Sample Web"
+weight: 2120
+---
+
+This section walks you through monitoring a sample web application. The application is instrumented with Prometheus Golang client in its code. Therefore, it can expose metrics directly without the help of exporters.
+
+## Prerequisites
+
+- Please make sure you enable the OpenPitrix system.
+- You need to create a workspace, a project, and a user account for this tutorial. This account needs to be a platform regular user and to be invited as a self provisioner. In this tutorial, we will work with the workspace `my-workspace` and the project `demo`. We will log in as `project-self-provisioner` and create the project `demo` beforehand.
+- You know Helm Chart.
+- You know [PromQL](https://prometheus.io/docs/prometheus/latest/querying/examples/).
```

Comment:
->
- Knowledge of Helm Chart and [PromQL](https://prometheus.io/docs/prometheus/latest/querying/examples/).

Reply:
done